### PR TITLE
Update docs - add on-chain payouts page

### PR DESCRIPTION
**Pull request by redphix**
## What does this PR do?

Add a page for on-chain payouts

## Checklist before merging
- [x] If it's a frontend feature, I have ran prettier `cd frontend; npm run format`. If it's a mobile app feature I ran `cd mobile; npm run format`.

### CODE REVIEW

File: docs/_pages/docs/03-understand/14-on-chain-payouts.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+something like this with the transaction id of the payout transaction:
+
+<div align="center">
+    <img src="/assets/images/understand/14-on-chain-payouts/successful-trade-on-chain.png"/>
+</div>
+
+### On-chain payout fees
+
+The on-chain payout fees (aka Swap fees) keeps changing from time to time. It
+may range from 1% to 10%. To get the current on-chain fees you can check the
```
Comment (Reckless-Satoshi) [inline docs/_pages/docs/03-understand/14-on-chain-payouts.md:49]: Still fluid, I think the bounds now are around ~0.8% to 5% .

File: docs/_data/navigation.yml [hunk 2]
```diff
@@ -44,6 +44,8 @@ docs:
         url: /docs/bonds/
       - title: '<span class="side-bar-item">&nbsp;&nbsp;<img id="side-icon-small" src="/assets/vector/money-bill-transfer.svg"/>Trade Escrow</span>'
         url: /docs/escrow/
+      - title: '<span class="side-bar-item">&nbsp;&nbsp;<img id="side-icon-small" src="/assets/vector/link-solid.svg"/>On-Chain Payouts</span>'
```
Comment (Reckless-Satoshi) [inline docs/_data/navigation.yml:47]: Cool, even added a new icon ^^

**Review by Reckless-Satoshi — APPROVED** [review]
Very nice and accurate write up! 

Looks good as is. Please submit a invoice from a proxy wallet for 20K Sats and a long expiration time

File: docs/_pages/docs/03-understand/14-on-chain-payouts.md (hunk 1 above)
Comment (redphix) [reply docs/_pages/docs/03-understand/14-on-chain-payouts.md:49]: Ah I see. I just referred to the sample-env file.

**Comment by redphix** [comment]
Thank you!
```
lnbc200u1p35grttpp5q7zann3c9dnwqxarqd6sv93vm8e5jdxrzqaf4ry67a429nqh3uwqdqqcqzpgxqyd9uqsp5f7vgttuw67m5jr9ddl7jfkc2mt96jyn6a0f9f0vpl6egkyqg7v3s9qyyssqc8g6e3tm3lyarde48cw9l50x60l4s037x4sv5at00q4cx5e5v9s36eyec8f86a80g2dtgwhy0e73tajlkfwgques8kl87wdgaljzqzsqy9tm0s
```

**Comment by Reckless-Satoshi** [comment]
> Thank you!
> 
> ```
> lnbc200u1p35grttpp5q7zann3c9dnwqxarqd6sv93vm8e5jdxrzqaf4ry67a429nqh3uwqdqqcqzpgxqyd9uqsp5f7vgttuw67m5jr9ddl7jfkc2mt96jyn6a0f9f0vpl6egkyqg7v3s9qyyssqc8g6e3tm3lyarde48cw9l50x60l4s037x4sv5at00q4cx5e5v9s36eyec8f86a80g2dtgwhy0e73tajlkfwgques8kl87wdgaljzqzsqy9tm0s
> ```

8b9a3f32183837e67ea4b3d78f316e04217fd51c723ec7e2c7206117a416df72

File: docs/_pages/docs/03-understand/14-on-chain-payouts.md (hunk 1 above)
Comment (Reckless-Satoshi) [reply docs/_pages/docs/03-understand/14-on-chain-payouts.md:49]: Maybe for future docs we should keep in mid to add these variables into https://github.com/Reckless-Satoshi/robosats/blob/936d19c6fcaf937f49f86036401fe91aeea2f396/docs/_config.yml#L32-L45 And include in the markdown file as ` {{site.robosats.VARIABLE_NAME}}`

**Pull request merged**